.changeset/quick-kids-collect.md

```diff
@@ -0,0 +1,6 @@
+---
+"@kaizen/draft-illustration": major
+---
+
+- Remove animated Spot illustrations, defaulting to Static illustrations.
+- Deprecated `isAnimated` prop on Spot illustrations
```

Comment:
Hmm if it's a major, I guess no point making it deprecated other than telling people why

Reply:
Yeah. That's why I said if we deprecate it, it should be minor lol.